applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.h

```diff
@@ -205,7 +204,6 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) ShockCapturingPhysicsBasedProcess :
     ///@name Private Operations
     ///@{
 
-    void ValidateAndAssignParameters(Parameters &rParameters);
```

Comment:
Why are you removing this function? It helps to maintain constructors clean and it might be useful if we eventually have more constructors.